lib/qa/authorities/linked_data/config.rb

```diff
@@ -28,7 +28,7 @@ def initialize(auth_name)
       end
 
       def search
-        @search ||= Qa::Authorities::LinkedData::SearchConfig.new(authority_config.fetch(:search), prefixes)
+        @search ||= Qa::Authorities::LinkedData::SearchConfig.new(authority_config.fetch(:search), prefixes, self)
       end
```

Comment:
passing self so search config has access to the full authority config for information like `authority_name`.